.backportrc.json

```
@@ -1,6 +1,6 @@
 {
   "upstream": "elastic/beats",
-  "branches": [ { "name": "7.x", "checked": true }, "7.10" ],
+  "branches": [ { "name": "7.x", "checked": true }, "7.11", "7.10" ],
```

With 7.11 then there will not more 7.10 releases?

```suggestion
  "branches": [ { "name": "7.x", "checked": true }, "7.11" ],
```

---

on Jan 5th we will release 7.10.2 so 7.10 is still a thing 😄